org.opent2t.sample.binaryswitch.superpopular/com.wink.binaryswitch/js/thingTranslator.js

```diff
@@ -166,12 +166,16 @@ class Translator {
         return postDeviceResource(di, 'power', payload);
     }
 
-    postSubscribe(callbackUrl, verificationRequest) {
-        return winkHub._subscribe(deviceType, deviceId, callbackUrl, verificationRequest);
+    postSubscribe(subscriptionInfo) {
+        subscriptionInfo.deviceId = deviceId;

```

Comment:
In wink translators is is deviceId (line 124).  Smarthings uses controlId.  We do want our internal naming to be consistent, but that requires changes to all the wink translators so I'll send that out in a later PR (it also doesn't affect anything so it won't matter for publishing to NPM).

---
In reply to: [92070474](https://github.com/openT2T/translators/pull/136#discussion_r92070474) [](ancestors = 92070474)